test_shaders.py

```diff
@@ -232,6 +232,8 @@ def cross_compile_msl(shader, spirv, opt, iterations, paths):
         msl_args.append('--emit-line-directives')
     if '.multiview.' in shader:
         msl_args.append('--msl-multiview')
+    if '.no-layered.' in shader:
+        msl_args.append('--msl-disable-layered-rendering')
```

Comment:
Sure?

Reply:
Shouldn't that be --msl-multiview-disable-layered-rendering?

Reply:
Oops. Fixed.